database/pdb.ksy

```diff
@@ -0,0 +1,961 @@
+meta:
+  id: pdb_file
```

Comment:
Should you use `rekordbox_pdb`?

Reply:
We are not certain that rekordbox is the  only program that creates files in this format. It is used by a number of embedded databases, evidently.

Reply:
Although, now that I think about it, only rekordbox would use this precise set of table and column structures, so yes, this would be a good change, I will make it. Thanks!

Reply:
All right, this is done; I also renamed `anlz_file` to `rekordbox_anlz` and updated my API documentation and Beat Link’s use of it.